hack/start-ec-task.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/bash
```

Comment:
could this be in hack/enterprise-contract/ folder?

Reply:
Yeah that makes sense, maybe a more demo focused version of it. I'd keep this here also though for use when hacking on the task definition.